de.metas.acct.base/src/main/java-legacy/org/compiere/acct/DocLine_Inventory.java

```diff
@@ -55,7 +58,12 @@ public DocLine_Inventory(final I_M_InventoryLine inventoryLine, final Doc_Invent
 			qty = qtyCount.subtract(qtyBook);
 		}
 
-		setQty(Quantity.of(qty, getProductStockingUOM()), false);
+		//calculate the cost price considering qty in inventory UOM
+		if (inventoryLine.getCostPrice() != null && qty.signum() > 0) {
```

Comment:
not a big deal for this issue, but for the next one, please make sure to synch your formatting settings with the settings that we have in the `metasfresh-dev` repo.
maybe there are intellij-settings to import. probably, @TheBestPessimist can help you